lib/Ohmbrewer_Sprouts.cpp

```diff
@@ -315,15 +318,18 @@ bool Ohmbrewer::Sprouts::arePinsInUse(std::list<int>* newPins) {
  * Parses a given string of characters into the pins for a Temperature Sensor
  * @param params The buffer to use for strtok'ing. This method will not delete the buffer!
  * @param busPin The bus pin
+ * @param index The onewire sensor index (-1 if unused / non onewire)
  * @return Error or success code, according to the requirements specified by addSprout
  */
-int Ohmbrewer::Sprouts::parseTemperatureSensorPins(char* params, int &busPin) {
-    String pin = String(strtok(NULL, ","));
+int Ohmbrewer::Sprouts::parseTemperatureSensorPins(char* params, int &busPin, int &index) {
```

Comment:
Refactor the name of the method if you'd like, but I think we shouldn't take the bus pin in here. We only support OneWire sensors at the moment, and we only support using a single pin for the bus. There's no real reason to pass it in every time. Instead, if we want to support changing the OneWire bus pin, we can add a new Particle.function for that. It's not like our OneWire library really supports running more than one bus at a time (if you want that) anyway - the function to set the pin is global.